dataland-frontend/src/utils/FileUploadUtils.ts

```diff
@@ -25,10 +25,15 @@ export async function uploadFiles(
   getKeycloakPromise: () => Promise<Keycloak>,
 ): Promise<void> {
   const documentControllerApi = new ApiClientProvider(getKeycloakPromise()).apiClients.documentController;
+  const alreadyUploadedFileReferences = new Set<string>();
   for (const fileToUpload of files) {
+    if (alreadyUploadedFileReferences.has(fileToUpload.fileReference)) {
```

Comment:
Why is this added?

Reply:
Maybe we can talk about this in a short call so i get a better understanding what was changed here.

Reply:
Of cours. However a short explanation: In a Create*DataModel.vue file we filtered for documents with the same has to not upload them twice and each of these used this function to upload the documents. The idea was before doing this filtering for each creation form we can do it at this centralized place such that noone must ever bother with it again.

Reply:
ok, thank you for your clarification